universal-login-web3/src/ui/utils/getValueFromSearch.ts

```diff
@@ -0,0 +1,11 @@
+import {MissingParameter} from './errors';
+
+export function getValueFromSearch(search: string, paramName: string) {
```

Comment:
What about having a function `parseSearch(serach: string): Record<string, string>` that will return an object with all keys and values parsed from search